REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.2-
 URIParser
-SHA
\ No newline at end of file
+SHA
+Compat
```

Comment:
I don't think we should bound the other version numbers just to get rid of warnings.  The version bounds should only be for things that are *required* to function.  I will fix the Compat version.